src/Stateless/Properties/AssemblyInfo.cs

```diff
@@ -1,21 +1,8 @@
-﻿using System;
-using System.Reflection;
-using System.Runtime.CompilerServices;
-using System.Resources;
-
-[assembly: AssemblyTitle("Stateless")]
-[assembly: AssemblyProduct("Stateless")]
-[assembly: AssemblyCopyright("Copyright © Stateless Contributors 2009-2016")]
-
-// This value will remain fixed throughout the 4.x series.
-[assembly: AssemblyVersion("4.0.0.0")]
```

Comment:
I’ll revert this small change to avoid chances of breaking anything-
 Can be considered separately